ingestion-beam/src/main/java/com/mozilla/telemetry/Republisher.java

```diff
@@ -90,6 +91,9 @@ public static PipelineResult run(RepublisherOptions.Parsed options) {
 
     // Republish to per-docType destinations.
     if (options.getPerDocTypeEnabledList() != null) {
+      // todo: this will be removed
+      decoded.apply(RepublishPerDocTypeEnabled.of(options));
```

Comment:
This will be removed once `--perDocTypeDestination` and `--perDocTypeEnabledList` are removed.